src/plots/cartesian/axes.js

```diff
@@ -1683,12 +1682,16 @@ axes.doTicks = function(gd, axid, skipTitle) {
                 counterAxis._length + 10 +
                     fontSize*(offsetBase + (ax.showticklabels ? 1.5 : 0.5)));
 
+            if(ax.rangeslider && ax.rangeslider.visible && ax._boundingBox) {
+                y += gs.h * ax.rangeslider.thickness + ax._boundingBox.height;
```

Comment:
Could you clarify?

I'll have to look again, but I think I tried to use `fullLayout._size`, but because it's height was being modified by the rangeslider, using it instead of calculating would create an infinite loop (where it would get scaled down infinitely by `1 - ax.rangeslider.thickness`.